### PR TITLE
Edit virtual-players-focus.md

### PR DESCRIPTION
**Pull request by amanda-butler-unity**
Edit page for style and accuracy.

#### **WARNING**: Please make your PR against the `develop` branch.
- - -
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**

**Issue Number:**

### CODE REVIEW

File: mppm/virtual-players/virtual-players-focus.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ To switch between virtual player views in the Multiplayer Play Mode window, use
 ## Focus on a virtual player in the Multiplayer Play Mode window {#focus-play-mode}
 
 1. Open the Multiplayer Play Mode window.
-2. In the Virtual Players section.
+2. Navigate to the Virtual Players section.
 3. Open a player's **More** (**⋮**) menu.
```
Comment (amanda-butler-unity) [inline mppm/virtual-players/virtual-players-focus.md:16]: Are steps 3 and 4 accurate? I couldn't find the More menu or any Focus on Player option in a dropdown.

File: mppm/virtual-players/virtual-players-focus.md [hunk 2]
```diff
@@ -31,4 +31,4 @@ Use the following keyboard shortcuts to switch focus to a specific virtual playe
 | **Ctrl**+**F9** (macOS: **Cmd**+**F9**)   |  Focus on **Player 1**.   |
```
Comment (amanda-butler-unity) [inline mppm/virtual-players/virtual-players-focus.md:31]: Are the keyboard commands accurate? I couldn't get them to work.

Comment (Vic-Cooper) [reply mppm/virtual-players/virtual-players-focus.md:31]: They should be accurate for this release, yes. These are new additions and I took these definitions straight from the code so they definitely should work!

File: mppm/virtual-players/virtual-players-focus.md (hunk 1 above)
Comment (Vic-Cooper) [reply mppm/virtual-players/virtual-players-focus.md:16]: this is accurate to the 1.0.0 release, yes :)

**Review by Vic-Cooper — APPROVED** [review]

**Pull request merged**